wwwroot/css/site.css

```diff
@@ -2,72 +2,119 @@
 for details on configuring this project to bundle and minify static web assets. */
 
 a.navbar-brand {
-  white-space: normal;
-  text-align: center;
-  word-break: break-all;
+    white-space: normal;
+    text-align: center;
+    word-break: break-all;
+    color: #000000;
 }
 
+
+.nav-pills .nav-link.active,
+.nav-pills .show>.nav-link {
+    color: #fff;
+    background-color: #183e74;
+    border-color: #183e74;
+}
+
+
 /* Provide sufficient contrast against white background */
+
 a {
-  color: #0366d6;
+    /* Links werden in CC grün hervorgehoben */
+    color: #183e74;
+    font-family: "Franklin Gothic";
+    
 }
 
-.btn-primary {
-  color: #fff;
-  background-color: #1b6ec2;
-  border-color: #1861ac;
+
+/* Button Styles
+-------------------------------- */
+.btn-link {
+    color: #183e74;
 }
 
-.nav-pills .nav-link.active, .nav-pills .show > .nav-link {
-  color: #fff;
-  background-color: #1b6ec2;
-  border-color: #1861ac;
+.btn-primary {
+    color: #fff;
+    background-color: #183e74;
+    border-color: #183e74;
 }
 
+
 /* Sticky footer styles
 -------------------------------------------------- */
+
 html {
-  font-size: 14px;
+    font-size: 14px;
 }
+
 @media (min-width: 768px) {
-  html {
-    font-size: 16px;
-  }
+    html {
+        font-size: 16px;
+    }
+}
+
+
+
+/* ============== TextFont Frankling Gothic  From (CSS FONT Generator) https://www.cssfontstack.com/franklin-gothic-medium */
+
+body {
+    /* Schriftart Franklin Gothik Book */
+    font-family: Franklin Gothic Medium, Franklin Gothic, ITC Franklin Gothic, Arial, sans-serif;
```

Comment:
Our main Website uses [Libre Franklin](https://fonts.google.com/specimen/Libre+Franklin), which looks good imo. I don't have any Franklin Gothic Font installed so I can't judge how it looks atm

Reply:
We should deliver the font with the website. Therefore you should import their files and reference them in the site.css (https://www.w3schools.com/cssref/css3_pr_font-face_rule.asp)

Reply:
used titillium now